specs/standards/connecting-to-tool-use/initial-connection/initial-connection.test.js

```diff
@@ -158,7 +157,7 @@ describe('Standards: Initial Connection', () => {
       });
 
       const wwwAuth = authResponse.headers.get('WWW-Authenticate');
-      const metadataMatch = wwwAuth.match(/resource_metadata="([^"]+)"/);
+      const metadataMatch = wwwAuth.match(/resource_metadata_url="([^"]+)"/);
```

Comment:
resource_metadata is standard

Reply:
Fixed (now sends a compliant message and matches on resource_metadata=)